README.md

```diff
@@ -17,94 +25,136 @@
 
 ### Chapter 2: Getting Started with Dash
 
- - Structure of a Dash app: data, layout, callback
- - Installation
+ - Structure of a Dash app
+ - Installation and setup
+ - Create your first Dash app
+ - Live updates and debugging
 
-### Chapter 3: Dash Components and Layout
+### Chapter 3: Dash Components and Layouts
 
- - Core components: dcc.dropdown, dcc.graph 
- - HTML components: html.div
  - Design the app layout
-
-### Chapter 4: Callback I- connecting two components
-
- - Callback Decorator: Input, Output, Component_id, Component_property
- - Callback function
+ - Linking a CSS file
+ - Dash Core components
+   - Markdown
+   - Dropdown
+   - Checkbox
+   - Slider
+   - Graph
+   - etc...
+ - Dash Bootstrap Components:
```

Comment:
I think we should divide this into two sections because dbc for layout is a big topic that separate from using dbc to add new components to app.
1. Dash Bootstrap Components for defining Layout.
2. Dash Bootstrap Components for additional components

Actually, I originally put DBC in part 3 because my goal was to focus on getting the reader to build their basic app and deploy it first. Then, in part 3, the reader could learn additional components and advanced layout with DBC. That way, we avoid frontloading so much information. What are your thoughts here?

Reply:
Ok I agree we can split up DBC into two components: first, to just use it as a way to create divs within a container (that is basic).

Then, later, some of the more complex aspects of DBC.

I think we should start their basic app with DBC because it will be a really nice app and we should de-emphasize `html`

Reply:
I agree about starting basic app with dbc and de-emphasizing html. Great point. 
So should we put the more complex aspects of DBC in part III or you want to keep it here?

Reply:
I just added "advanced DBC" to Part 3, resolving this for now.